src/gui/DatabaseSettingsWidgetGeneral.ui

```diff
@@ -134,6 +134,32 @@
      </layout>
     </widget>
    </item>
+   <item>
+    <widget class="QGroupBox" name="groupBox_3">
+     <property name="title">
+      <string>Compression</string>
+     </property>
+     <layout class="QVBoxLayout" name="verticalLayout_2">
+      <item>
+       <widget class="QRadioButton" name="compressionGZip">
+        <property name="text">
+         <string>GZip (recommended)</string>
```

Comment:
Recommend rename this to "Compressed (GZip)". Its recommended by virtue of being defaulted to compressed.

Reply:
I don't think that goes well with the context. "compressed" is not a "compression" (which is what the group box is called). Also the other option would have to be "uncompressed" then, which is also not a compression method and looks too similar to "compressed" IMHO.

Reply:
I only say that because most users (especially on windows) have no idea what gzip is. After teaching my mother to use this software I realized we have to be careful about being too complex.

Reply:
True, but on the other hand it's the wording KeePass uses. I don't really think it matters here how we call it, non-technical users will probably not understand it in any case and simply leave it as is if it is "recommended".

Reply:
I was actually thinking about adding another algorithm, but the only thing we have in code is zlib. Everything else would have required an external lib.

Reply:
@phoerious Other algorithms may be supported by using KF5Archive.